lib/auth/clt.go

```diff
@@ -1567,10 +1567,31 @@ func (c *Client) grpcGetUsers(withSecrets bool) ([]services.User, error) {
 	return users, nil
 }
 
-// DeleteUser deletes a user by username
-func (c *Client) DeleteUser(user string) error {
-	_, err := c.Delete(c.Endpoint("users", user))
-	return trace.Wrap(err)
+// DeleteUser deletes a user by username.
+func (c *Client) DeleteUser(ctx context.Context, user string) error {
+	clt, err := c.grpc()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	req := &proto.DeleteUserRequest{Name: user}
+	_, grpcErr := clt.DeleteUser(ctx, req)
```

Comment:
change `grpcErr` to `err` unless it needs to be different

Reply:
revised